conf/application.conf

```diff
@@ -65,6 +65,12 @@ microservice {
       host = localhost
       port = 8100
     }
+
+    identity-verification {
+      protocol = http
+      host = localhost
+      port = 9927
+    }
```

Comment:
Is there another PR to override this in app-config-base? I assume we'd want that merged first too.